.circleci/config.yml

```diff
@@ -175,13 +171,12 @@ jobs:
       - *cache_save_git
       - *cache_restore_bundler
       - *set_ruby
-      - run:
-          name: Setup Build
-          command: |
-            mkdir -p ~/test-reports
-            brew update # Needed because this lane uses "brew bundle" and CircleCI's brew install is too old for that
       - *bundle_install
       - *cache_save_bundler
+      - run:
+          name: debug | brew version
+          command: |
+            brew -v
```

Comment:
However, fastlane.swift uses swiftformat installed via `brew bundle`. This would need this debug info.